packages/sync-actions/test/cart-discounts-sync.spec.js

```diff
@@ -332,4 +332,68 @@ describe('Cart Discounts Actions', () => {
     const actual = cartDiscountsSync.buildActions(now, before)
     expect(actual).toEqual(expected)
   })
+
+  describe('custom fields', () => {
+    it('should build `setCustomType` action', () => {
```

Comment:
A test for changing multiple fields is missing

Reply:
That test is on the custom field spec itself. This is just meant - as for categories and discount codes too - to be a rather thin integration test. I saw two options

1. Mocking 
2. Thin integration test

I felt mocking is less obvious to know what happens once you change the implementation.